content/events-api/overview.md

```diff
@@ -2,6 +2,22 @@
 
 _All the essential things you need to know._
 
-## Scope of the Events API
+By using the Akeneo Events API, you will receive events on: product creation, product update, and product deletion. 
+The feature is working on simple products and product models and variants. 
 
-// todo
\ No newline at end of file
+::: info 
+For more information, please read our help center article [What is the event subscription?](https://help.akeneo.com/pim/serenity/articles/what-is-the-event-subscription.html) to discover which type of events the Akeneo PIM triggers and in what cases. 
+It explains to the Akeneo PIM users how to setup their PIM to use an extension that uses events to synchronize the Akeneo PIM product data with other applications. 
+::: 
+
+## The event loop
```

Comment:
```suggestion
## The event API loop
```

We should take care of the words we are using everywhere (event, event API, event subscription, etc.).